CHANGELOG.md

```diff
@@ -17,7 +17,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Fixed
 
--
+- Fix "ripple" error
```

Comment:
```suggestion
- 
```

This one actually doesn't have to go in the changelog, because it's not something the user would see or notice a difference in.

Reply:
ok fixed!